docs/tutorials/getting_started/customize_your_deployment.md

```diff
@@ -69,16 +69,13 @@ Great Expectations allows you to connect to data in a wide variety of sources, a
 
 * [How to connect to a Athena database](/docs/guides/connecting_to_your_data/database/athena)
 * [How to connect to a BigQuery database](/docs/guides/connecting_to_your_data/database/bigquery)
-* [How to connect to a MSSQL database](/docs/guides/connecting_to_your_data/database/mssql)
```

Comment:
This is currently an open PR.. shouldn't be removed 
https://github.com/great-expectations/great_expectations/pull/3353

Reply:
Ok.. Added that back